ui/src/config/section/infra/secondaryStorages.js

```diff
@@ -51,9 +52,19 @@ export default {
     component: () => import('@/components/view/AnnotationsTab.vue')
   }],
   actions: [
+    {
+      api: 'migrateSecondaryStorageData',
+      icon: 'drag',
+      shortKey: ['m'],
+      label: 'label.migrate.data.from.image.store',
+      listView: true,
+      popup: true,
+      component: () => import('@/views/infra/MigrateData.vue')
+    },
```

Comment:
@apurv-1 is this part of this PR?

Reply:
I think I might have added the shortKey on the object but then It was removed(Since its here in this [PR #5103](https://github.com/apache/cloudstack/pull/5103/files#diff-03295e69be9bb1aec7bb1a7865447db525533f3ae3ef2fe19c5d2942312c50c2R64)).
But no worries I'll remove it. : )